jekyll/_cci2/gitlab-integration.adoc

```diff
@@ -194,6 +240,15 @@ Organization administrator as well as organization contributor roles are current
 +
 If they do not currently have a CircleCI account, they will need to sign up. If they already have a CircleCI account, they are added to the organization, and if they are logged in, they will see the organization as an option in the organization switcher in the top left corner of the web app.
 
+[#organization-settings-integrations]
+=== Integrations (GitLab Self-managed only)
+
+For GitLab Self-managed organizations, you may connect additional self-managed instances to be integrated with your organization. Navigate to **Integrations** section within **Organization Settings** to add a new instance. You will need to enter the instance URL, your personal access token, and the SSH key fingerprint as described in the xref:#sign-up[Sign up] section earlier in this document.
```

Comment:
We won't be asking for a personal access token on this page. 

Based on the current design
If users have already added the `known_hosts` then they will be able to add an instance URL. [Designs](https://www.figma.com/file/yWoX7C7KShtPaA5BgxXSLK/GL-self-managed-connection?node-id=658-11928&t=PdGGo884VLk8Xnpn-0)

<img width="659" alt="image" src="https://user-images.githubusercontent.com/22925321/229543643-8e985bfa-3346-428e-8518-1818951cf64a.png">

<img width="540" alt="image" src="https://user-images.githubusercontent.com/22925321/229541480-e44e61bc-6d0d-4385-a244-7fc1091fe8b3.png">.

If they have not added the `known_hosts` and have no connections they will see a `Set up Integration` button which will open a modal that will let them add the `known_hosts`. [Designs](https://www.figma.com/file/yWoX7C7KShtPaA5BgxXSLK/GL-self-managed-connection?node-id=652-10380&t=PdGGo884VLk8Xnpn-4)

<img width="683" alt="image" src="https://user-images.githubusercontent.com/22925321/229543071-a014e5c3-8d00-437e-8e79-8e5423ec8749.png">

<img width="312" alt="image" src="https://user-images.githubusercontent.com/22925321/229543120-33a7e3c6-5588-4b29-90b0-93e175c3685f.png">